server/app/routes/v1/users_api.rb

```diff
@@ -8,6 +8,24 @@ class UsersApi < Roda
 
     route do |r|
       r.post do
+        r.on 'roles' do
+          validate_access_token
+          require_current_user
+
+          params = parse_json_body          
+          params[:current_user] = current_user
+          params[:role_name] = params['role']
```

Comment:
Any reason for this? Should we just pick a good name for the "role" attribute and use that in every place.